docs/B-Computations/logic.md

```diff
@@ -391,7 +395,8 @@ for (slices = 4; slices > 0; --slices)
 
 Flagging is a method of coding iteration constructs within the single-entry single-exit rule of structured programming.  Consider the flow-chart on the left side in the figure below.  This design contains a path that crosses another path. 
 
-![](https://ict.senecacollege.ca//~ipc144/pages/images/flags.png)
+![flags](/img/image37.png)
```

Comment:
Suggestion to provide meaningful alt text. Example: `Spaghetti code versus Structured code flow charts`